Workshop/nodegoat/.whitesource_init_pull_request

```diff
@@ -5,20 +5,35 @@
     "projectToken": "",
     "baseBranches": [],
     "enableLicenseViolations": "true"
+  },
+    "scanSettingsSAST": {
```

Comment:
What happen if SAST is not enabled but the setting is? Would it show broken SAST scan?

Reply:
yes - the check run would appear but fail. Users should not enable, but I wanted to leave it as it is part of the default .whitesource configuration file.

Reply:
It will make things complicated and less professional.
Let's create another one with SAST enabled